packages/@uppy/transloadit/src/Client.js

```diff
@@ -96,5 +103,41 @@ module.exports = class Client {
   getAssemblyStatus (url) {
     return fetch(url)
       .then((response) => response.json())
+      .catch((err) => this._reportError(err, { url }))
+  }
+
+  submitError (err, { endpoint, instance, assembly }) {
+    return fetch('https://status.transloadit.com/client_error', {
+      method: 'post',
+      body: JSON.stringify({
+        endpoint,
+        instance,
+        assembly_id: assembly,
+        ip: null, // can't this just be done by statuspage?
+        agent: typeof navigator !== 'undefined' ? navigator.userAgent : '',
+        error: err.message
```

Comment:
do we only have a string error, no type or constant or other payload that we can use to more easily identify them?

Reply:
it's what i took from the jQuery SDK, we could do `error: err.message` and `type: 'TUS_ERROR'`/`type: 'API_ERROR'` or so though

Reply:
That'd be a nice addition, if it's easy to add? After that i think this is good to go!